src/main/java/net/sf/jabref/logic/fetcher/IEEE.java

```diff
@@ -0,0 +1,55 @@
+package net.sf.jabref.logic.fetcher;
+
+import java.io.IOException;
+import java.net.URI;
+import java.net.URL;
+import java.util.Objects;
+import java.util.Optional;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+
+import com.google.common.base.Charsets;
+
+import net.sf.jabref.logic.net.URLDownload;
+import net.sf.jabref.logic.util.DOI;
+import net.sf.jabref.model.entry.BibEntry;
+
+public class IEEE implements FullTextFinder {
+
+    private static final Log LOGGER = LogFactory.getLog(IEEE.class);
+    private static final Pattern STAMP_PATTERN = Pattern.compile("(/stamp/stamp.jsp\\?tp=&arnumber=[0-9]+)");
+    private static final Pattern PDF_PATTERN = Pattern
+            .compile("\"(http://ieeexplore.ieee.org/ielx[0-9/]+\\.pdf[^\"]+)\"");
+    private static final String IEEE_DOI = "10.1109";
+    private static final String BASE_URL = "http://ieeexplore.ieee.org";
+
+
+    @Override
+    public Optional<URL> findFullText(BibEntry entry) throws IOException {
+        Objects.requireNonNull(entry);
+        Optional<URL> pdfLink = Optional.empty();
```

Comment:
The local variable is actually not needed, just return the appropriate value at the right place.